scipy/sparse/linalg/eigen/_svds.py

```diff
@@ -216,18 +300,44 @@ def matmat_XH_X(x):
         if k == 1 and v0 is not None:
             X = np.reshape(v0, (-1, 1))
         else:
-            X = np.random.RandomState(52).randn(min(A.shape), k)
+            if rs_was_None:
+                X = np.random.RandomState(52).randn(min(A.shape), k)
```

Comment:
Do we want to maintain the original behavior of `lobpcg` here?

Reply:
I'll probably have to rely on you to explain what has changed.  I'm not sure what the push-back will be to changing some of these functions will be (even if it's for the better)

Reply:
Originally it used `np.random.RandomState(52)` for the random number generator. This was never a good idea; it should have accepted `random_state`, and if not it should have used `np.random.rand` so that the user could control the behavior with `np.random.seed`, I think. 
Now that we're adding `random_state` as a parameter, it is going to look really strange that `RandomState(52)` is going to be used if the user doesn't specify a `random_state`.

Reply:
Ah, got it.  Yes, let's bring this up in the review